### PR TITLE
Add redux connect slides with example and update vanilla redux

### PR DESCRIPTION
**Pull request by kulkarniankita**
1st PR, there are still some gaps in the redux content but before I worked on it further, I wanted to create this PR and get some review 😄

### CODE REVIEW

File: src/slides/redux-intro/reducer-example.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const visibilityFilter = (state = "SHOW_ALL", action) => {
 const todos = (state = [], action) => {
   switch (action.type) {
     case "ADD_TODO":
-      return state.concat([{ text: action.text, completed: false }]);
+      return state.todos.concat([{ text: action.text, completed: false }]);
```
Comment (kulkarniankita) [inline src/slides/redux-intro/reducer-example.js]: Should this be `todos`?

Comment (pkanal) [reply src/slides/redux-intro/reducer-example.js]: i think its just state because we're using `combineReducers`

Comment (kulkarniankita) [reply src/slides/redux-intro/reducer-example.js]: Good point. I will fix it.

**Comment by kulkarniankita** [comment]
@pkanal Thanks for updating it new commits ✨ ⭐️

**Pull request merged**